src/custom/state/claim/actions.ts

```diff
@@ -34,7 +35,7 @@ export const setActiveClaimAccountENS = createAction<string>('claim/setActiveCla
 export const setIsSearchUsed = createAction<boolean>('claim/setIsSearchUsed')
 
 // claiming
-export const setClaimedAmount = createAction<number>('claim/setClaimedAmount')
+export const setClaimedAmount = createAction<CurrencyAmount<Token> | null>('claim/setClaimedAmount')
```

Comment:
we don't save these complex numbers in state unserialised. just save the atom amount.
assuming 
```const currencyAmount: CurrencyAmount<Currency>```
then save it as
```currencyAmount.quotient.toString()```